src/com/gfredericks/test/chuck/properties.cljc

```diff
@@ -81,8 +80,8 @@
   [bindings expr]
   (let [bound-names (for-bindings bindings)
         quoted-names (map #(list 'quote %) bound-names)]
-    `(prop/for-all [{:syms [~@bound-names]}
-                    (gen'/for ~bindings
+    `(clojure.test.check.properties/for-all [{:syms [~@bound-names]}
+                    (com.gfredericks.test.chuck.generators/for ~bindings
```

Comment:
Yes, I changed this and a couple more places where an alias could be used in 5ad5d11